packages/design-system/src/components/ChoiceList/Choice.tsx

```diff
@@ -76,7 +76,7 @@ export interface ChoiceProps {
    */
   name: string;
   onBlur?: (...args: any[]) => any;
-  onChange?: (...args: any[]) => any;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => any;
```

Comment:
Does this have a regression risk? Changing the type from `any` to something else more specifically types seems like it might?

Reply:
We're preferring correctness over risk of breaking changes in these conversions. One set of breaking changes at once